intro-HPC/ch_troubleshooting.tex

```diff
@@ -29,6 +29,22 @@ \section{Out of quota issues}
 \fi
 % FIXME: Add how to request quota section
 
+\ifwindows
```

Comment:
We should probably drop this guard, since users may somehow get their hands on files that they didn't upload themselves from a Windows system (e.g. got them from a colleague, or downloaded from the web, etc.)